app/schemas/io.github.droidkaigi.confsched2018.data.db.AppDatabase/6.json

```diff
@@ -2,7 +2,7 @@
   "formatVersion": 1,
```

Comment:
Please revert 6.json 🙇 
You can copy master 6.json
https://github.com/youta1119/conference-app-2018/blob/master/app/schemas/io.github.droidkaigi.confsched2018.data.db.AppDatabase/6.json